src/components/FiltersPanel/FiltersPanel.tsx

```diff
@@ -33,7 +33,14 @@ import { WatchingFilter } from '../WatchingFilter/WatchingFilter';
 import { tr } from './FiltersPanel.i18n';
 
 const take = 5;
-const queryParams = {
+
+// disable refetchOnMount since we use filtersPanelSsrInit
+const useQueryOptionsRefetchOnMount = {
+    refetchOnMount: false,
+};
+
+const useQueryOptions = {
+    ...useQueryOptionsRefetchOnMount,
```

Comment:
why can't you add `refectOnMount` here without a new variable?

Reply:
To use the same value (w/o copy-paste) for different hooks as options: [useQueryOptions](https://github.com/taskany-inc/issues/pull/1395/files#diff-87ff408285e4f0631d3338be356fce4cd02d84f8cbc7fbe3967af3eb453a9e2eR140) and [useQueryOptionsRefetchOnMount](https://github.com/taskany-inc/issues/pull/1395/files#diff-87ff408285e4f0631d3338be356fce4cd02d84f8cbc7fbe3967af3eb453a9e2eR187).